### PR TITLE
Add arm64 arch and NXP ls2080ardb platform

### PR DESCRIPTION
**Pull request by shengzhou**
Prerequisites:
 1. to build arm64 arch in ONL, it needs to use new builder8:1.5 
 2. it needs to first apply pull request floodlight/infra#54 for submodule infra build

 These patches passed build and run fine on nxp arm64 ls2080ardb.

This pull request is the second version:
1. rebase to the latest source (commit 43183f7c50de, Aug 15)
2. minor fix to address Jeff's comments.

### CODE REVIEW

**Comment by jnealtowns** [comment]
@shengzhou The pull request does not currently build because the buildroot toolchain is missing from the new builder8:1.5 image. The preconfigured toolchain settings in the buildroot.config-arm64 look like this:

    BR2_TOOLCHAIN_EXTERNAL=y
    # BR2_TOOLCHAIN_EXTERNAL_LINARO_AARCH64_13_01 is not set
    # BR2_TOOLCHAIN_EXTERNAL_LINARO_AARCH64_12_12 is not set
    # BR2_TOOLCHAIN_EXTERNAL_LINARO_AARCH64_12_11 is not set
    BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
    # BR2_TOOLCHAIN_EXTERNAL_DOWNLOAD is not set
    BR2_TOOLCHAIN_EXTERNAL_PREINSTALLED=y 
    BR2_TOOLCHAIN_EXTERNAL_PATH="/usr/buildroot/toolchains/arm64"
    BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="aarch64-buildroot-linux-gnu"
    BR2_TOOLCHAIN_EXTERNAL_PREFIX="aarch64-buildroot-linux-gnu" 
    BR2_TOOLCHAIN_EXTERNAL_GLIBC=y
    # BR2_TOOLCHAIN_EXTERNAL_CUSTOM_UCLIBC is not set
    BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
    BR2_TOOLCHAIN_EXTERNAL_INET_RPC=y
    # BR2_TOOLCHAIN_EXTERNAL_CXX is not set
    BR2_TOOLCHAIN_EXTRA_EXTERNAL_LIBS=""
    # BR2_TOOLCHAIN_EXTERNAL_GDB_SERVER_COPY is not set 
    BR2_TOOLCHAIN_HAS_NATIVE_RPC=y
    BR2_TOOLCHAIN_HAS_THREADS=y
    BR2_TOOLCHAIN_HAS_THREADS_DEBUG=y
    BR2_TOOLCHAIN_HAS_THREADS_DEBUG_IF_NEEDED=y
    BR2_TOOLCHAIN_HAS_SHADOW_PASSWORDS=y

Which toolchain was your environment setup to use?

File: tools/onlrfs.py [hunk 1]
```diff
@@ -235,6 +235,7 @@ class OnlRfsBuilder(object):
     MULTISTRAP='/usr/sbin/multistrap'
     QEMU_PPC='/usr/bin/qemu-ppc-static'
     QEMU_ARM='/usr/bin/qemu-arm-static'
+    QEMU_ARM64='/usr/bin/qemu-aarch64-static'
```
Comment (jnealtowns) [inline tools/onlrfs.py]: This binary is not available in the builder8:1.5 image.

**Comment by shengzhou** [comment]
@jnealtowns  You need to pull my builder8:1.5 to get the arm64 buildroot toolchain and qemu-aarch64-static  as below:

1. download my builder8:1.5 by command "docker pull shengzhou/builder8:1.5"

2. copy  /usr/buildroot/toolchains/arm64  and  /usr/bin/qemu-aarch64-static from shengzhou/builder8:1.5 to local builder8:1.5 
    $ docker exec a6ae2c764fae ls -al /usr/buildroot/toolchains/arm64 
    a6ae2c764fae
    total 32
    drwxr-xr-x  8 root root 4096 Aug  3 18:47 .
    drwxr-xr-x  6 root root 4096 Aug  3 18:46 ..
    drwxr-xr-x  6 root root 4096 Aug  2 18:22 aarch64-buildroot-linux-gnu
    drwxr-xr-x  2 root root 4096 Aug  2 18:22 bin
    drwxr-xr-x  6 root root 4096 Aug  2 18:22 include
    drwxr-xr-x  9 root root 4096 Aug  2 18:22 lib
    drwxr-xr-x  4 root root 4096 Aug  2 18:22 libexec
    drwxr-xr-x 19 root root 4096 Aug  2 18:22 share

    $ docker exec a6ae2c764fae ls -al /usr/bin/qemu-aarch64-static
    a6ae2c764fae
    -rwxr-xr-x 1 root root 3143680 Aug  3 18:33 /usr/bin/qemu-aarch64-static
    
3. push local new builder8:1.5 to dockerhub 
     $ docker push opennetworklinux/builder8:1.5

**Comment by shengzhou** [comment]
Today, I rebased  this pull request to the latest source (commit  43183f7 , Aug 15), tested latest image, it worked fine on arm64 ls2080ardb.

**Comment by jnealtowns** [comment]
@shengzhou  Thanks. I have pulled the assets from your docker image and will update the new 1.5  docker file to reproduce the image.

**Pull request merged**